discovery-provider/src/queries/search_queries.py

```diff
@@ -459,8 +477,13 @@ def track_search_query(session, searchStr, limit, offset, personalized, is_auto_
             Track.stem_of == None,
             Track.track_id.in_(track_ids),
         )
-        .all()
     )
+
+    # Filter to downloadable if needed
```

Comment:
this filter is redundant right?

Reply:
you're right it totally is - left over from when I was trying a different approach. nice catch! will remove